common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -1976,9 +1976,13 @@ def test_clone_xml_split(self):
             dest_store = self.store._get_modulestore_by_type(ModuleStoreEnum.Type.split)
             self.assertCoursesEqual(source_store, source_course_key, dest_store, dest_course_id)
 
-    @skip("PLAT-449 XModule TestMixedModuleStore intermittent test failure")
```

Comment:
Not yet sure that it is resolved... I haven't seen the problem, so I don't _know_ for sure that it is resolved yet...